charts/virtualcluster/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
```

Comment:
I really think this needs its own repo, so it can move at its own pace, you'll have to update the application spec every release!  I/you can use uni-helm-cluster-api as a template of how to do it.  It's literally a 30 minute job.